Optionally sign initial SOA query (#9672)
certbot/certbot: certbot/CHANGELOG.md

### Changed

* Optionally sign the SOA query for dns-rfc2136, to help resolve problems with split-view
  DNS setups and hidden primary setups.
  * Certbot versions prior to v1.32.0 did not sign queries with the specified TSIG key
    resulting in difficulty with split-horizon implementations.

>   * Certbot versions later than v1.32.0 signed queries by default, potentially causing

Reviewer: ```suggestion
  * Certbot v1.32.0 through v2.5.0 signed queries by default, potentially causing
```

    incompatibility with hidden primary setups with `allow-update-forwarding` enabled
    if the secondary did not also have the TSIG key within its config.

>   * Certbot versions later than v2.6.0 now do not sign queries by default, but allow

Reviewer: ```suggestion
  * Certbot v2.6.0 and later no longer signs queries by default, but allows
```

### Changed

> * Optionally sign the SOA query for dns-rfc2136, to help resolve problems with split-view

Author: Added some more detail here.
